### DISCUSSION TITLE
update libmesh fails

### DISCUSSION THREAD

**Discussion by srinath-chakravarthy** [idaholab/moose#21114] [Q&A Getting Started] [answered]
Hi All, 
I have moose installed on my HPC system and after updating moose via git, libmesh fails to compile.   I am using the manual hpc instructions to compile as the HPC does not have access to the internet. i am also attaching my config.log. I am on the latest master commit commit 16ca78fa2eefeec91b61b854a92b1ab22f6156e8 of moose. 
CXX      src/parallel/libmesh_opt_la-parallel_elem.lo
  CXX      src/parallel/libmesh_opt_la-parallel_ghost_sync.lo
  CXX      src/parallel/libmesh_opt_la-parallel_histogram.lo
  CXX      src/parallel/libmesh_opt_la-parallel_node.lo
  CXX      src/parallel/libmesh_opt_la-parallel_sort.lo
../src/mesh/poly2tri_triangulator.C:37:10: fatal error: poly2tri/poly2tri.h: No such file or directory
   37 | #include "poly2tri/poly2tri.h"
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[1]: *** [Makefile:27416: src/mesh/libmesh_opt_la-poly2tri_triangulator.lo] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/home/srinathcs/github/moose/libmesh/build'
make: *** [Makefile:32937: all-recursive] Error 1
[config.log](https://github.com/idaholab/moose/files/8763784/config.log)

**Comment by GiudGiud** [comment, marked as answer]
@roystgnr 
This is a new dependency of ours. Can you remove the libmesh folder and try again?

**Reply by roystgnr** [reply]
It looks as if the new poly2tri submodule isn't there.  A recursive submodule update might get it ... but yeah, removing libMesh and starting over from there might be safest.

**Comment by srinath-chakravarthy** [comment]
I'm confused, I did perform a recursive update. So to confirm, remove
libmesh and then run update script ??

**Reply by srinath-chakravarthy** [reply]
Ok removing libmesh and running the update script works. Thanks a lot.
